cli_reference/admin_cli_operations.adoc

```diff
@@ -17,6 +17,12 @@ syntax. You must xref:get_started_cli.adoc#cli-reference-get-started-cli[setup a
 you can perform these operations.
 
 ifdef::openshift-origin,openshift-enterprise[]
+The `oc` command provides command line tools for managing applications, while
+the `oadm` command provides tools for managing a cluster. The `openshift` binary
+is an all-in-one command that can be used for starting services that make up the
+{product-title} cluster. The `openshift` binary can perform all the same actions
+as the `oc` and `oadm` commands.
+
```

Comment:
This confuses me a bit. the binary can "be used for starting services" and perform all the same actions...
Is this redundant? Or, am I mistaken....
"The `openshift` binary is an all-in-one command that can perform all the same actions as the `oc` and `oadm` commands, such as starting services."